parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -291,6 +293,8 @@ try
     icacls.exe "c:\k"
     Get-ProvisioningScripts
     Get-LogCollectionScripts
+
+    Disable-KubeletServingCertificateRotationForTags

```

Comment:
This will fail since Disable-KubeletServingCertificateRotationForTags cannot be found in the current windows cse script package.

1. Update the scripts under staging/cse/windows
2. Release a new cse script package
3. Update the cse package version and add this line